packages/core/src/fields/inputFields/fields/ProgressBar/ProgressBarComponent.svelte

```diff
@@ -125,9 +125,12 @@
 		aria-valuemin={props.minValue}
 		aria-valuemax={props.maxValue}
 		aria-valuenow={value}
+		aria-label={value}
```

Comment:
I experimented a bit with with switching to `aria-labeledby` in case the labels are shown, but could get it to work. also this required a unique ID on the `mb-progress-bar-value` which adds unnecessary overhead